.github/actions/apt-x64/action.yml

```diff
@@ -0,0 +1,68 @@
+name: apt
+runs:
+  using: composite
+  steps:
+    - shell: bash
+      run: |
+        set -x
+        sudo apt-get update
```

Comment:
is `sudo` really needed here?

Reply:
Seems so. https://stackoverflow.com/questions/57982945/how-to-apt-get-install-in-a-github-actions-workflow